--- src/Inventor/Xt/Makefile.in	Mon Feb  7 22:12:59 2000
+++ src/Inventor/Xt/Makefile.in.corrected	Mon Feb  7 22:16:18 2000
@@ -173,7 +173,8 @@
 HEADERS =  $(libSoXtinc_HEADERS)
 
 DIST_COMMON =  $(libSoXtinc_HEADERS) Makefile.am Makefile.in \
-SoXtRenderArea.cpp.in SoXtRenderArea.h.in
+$(top_srcdir)/src/Inventor/Xt/common/SoAnyRenderArea.cpp.in \
+$(top_srcdir)/src/Inventor/Xt/common/SoAnyRenderArea.h.in
 
 
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
@@ -196,9 +197,9 @@
 	cd $(top_builddir) \
 	  && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
-SoXtRenderArea.h: $(top_builddir)/config.status SoXtRenderArea.h.in
+SoXtRenderArea.h: $(top_builddir)/config.status $(top_srcdir)/src/Inventor/Xt/common/SoAnyRenderArea.h.in
 	cd $(top_builddir) && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
-SoXtRenderArea.cpp: $(top_builddir)/config.status SoXtRenderArea.cpp.in
+SoXtRenderArea.cpp: $(top_builddir)/config.status $(top_srcdir)/src/Inventor/Xt/common/SoAnyRenderArea.cpp.in
 	cd $(top_builddir) && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
 mostlyclean-libLTLIBRARIES:
